### DISCUSSION TITLE
Ran `./automatic.sh install` but it says pytorch without cuda is installed and throws errors at launch

### DISCUSSION THREAD

**Discussion by ScionoicS** [vladmandic/sdnext#50] [Q&A]
This looks like an excellent webui, but i can't seem to troubleshoot this problem

**Comment by vladmandic** [comment]
can you copy&paste actual error?
if torch says no cuda or gpu detected, thats a low-level error, unlikely related to repo itself. but i'd be happy to help troubleshoot.
and sorry for late reply.
